test/dummy/app/models/custom_primary_key_record.rb

```diff
@@ -0,0 +1,11 @@
+require "securerandom"
+class CustomPrimaryKeyRecord < ActiveRecord::Base
+  self.primary_key = :uuid
+
+  has_paper_trail class_name: "CustomPrimaryKeyRecordVersion"
+  default_scope -> { where(name: "custom_primary_key_record") }
```

Comment:
it is to prevent the record from being returned on `version.item`. this bug surfaces in the `else` path of https://github.com/airblade/paper_trail/blob/v5.2.2/lib/paper_trail/reifier.rb#L25-L40

`version.item` returns nil when the item is destroyed, and when a default scope on the model prevents the association from returning it. I added this default scope to test both of those cases.